code/controlconfig/controlsconfig.cpp

```diff
@@ -393,21 +393,10 @@ int control_config_detect_axis()
 
 void control_config_conflict_check()
 {
-	int i, j, shift = -1, alt = -1;
-
+	int i, j;
+	
 	for (i=0; i<CCFG_MAX; i++) {
 		Conflicts[i].key = Conflicts[i].joy = -1;
-		switch (Control_config[i].key_id) {
-			case KEY_LSHIFT:
-			case KEY_RSHIFT:
-				shift = i;
```

Comment:
Sorry, my only context for that comment was a git grep where I saw that both variables were present in both files.  If the code surrounding them is significantly different then there probably is not much relation between them.